ReactAndroid/src/main/java/com/facebook/react/views/art/ARTShapeShadowNode.java

```diff
@@ -182,6 +234,21 @@ protected boolean setupFillPaint(Paint paint, float opacity) {
               (int) (mFillColor[2] * 255),
               (int) (mFillColor[3] * 255));
           break;
+        case 1:
+          int stopsCount = getStopsCount(mFillColor);
+          int [] stopsColors = new int [stopsCount];
+          float [] stops = new float[stopsCount];
+          gradientStopsParsing(mFillColor, stopsCount, stops, stopsColors);
+          paint.setShader(
+            new LinearGradient(
+              mFillColor[3] * 2,
```

Comment:
no this code not compensate screen density. i test it with IOS and Android. and on IOS intensity larger in 2 times. I can remove this but result on ios and android will be different))